### PR TITLE
python3Packages.pre-commit: fix hook-tmpl

### PR DESCRIPTION
**Pull request by cpcloud**
###### Motivation for this change

This PR fixes an issue where the latest pre-commit v2.16.0
generates a bash template that doesn't work with nix's Pythons.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by dotlambda — COMMENTED** [review]
Please move pre-commit out of `python-packages.nix`, it's an application and not a module.

**Comment by cpcloud** [comment]
@dotlambda Done.

**Review by dotlambda — CHANGES_REQUESTED** [review]
pre-commit shouldn't be in wlc's propagatedBuildInputs anyway, it's not in https://github.com/WeblateOrg/wlc/blob/main/requirements.txt

**Comment by cpcloud** [comment]
@dotlambda Can this be merged?

File: pkgs/tools/misc/wlc/default.nix [hunk 1]
```diff
@@ -1,6 +1,4 @@
-{ lib
-, python3
-}:
```
Comment (dotlambda) [inline pkgs/tools/misc/wlc/default.nix:3]: no need to change this

**Comment by dotlambda** [comment]
Please squash this down to 3 commits, then it's good to go I think.

**Comment by cpcloud** [comment]
@dotlambda Squashed, ready to go.

**Pull request merged**